### PR TITLE
chore(website): add UX guidelines base structure

### PR DESCRIPTION
**Pull request by craicoverflow**
- chore: use root 'docs' folder as website source
- chore: add UX guidelines base structure
- chore: use custom website sidebar

## Verification

From the `website` folder run `yarn start` to run the website locally.

### CODE REVIEW

File: docs/commands/_category_.json [hunk 1]
```diff
@@ -1,4 +0,0 @@
-{
```
Comment (wtrocki) [inline docs/commands/_category_.json:1]: [Question] Why - better to autogenerate that?

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
Amazing work! Let's not publish that yet as we do not have content

File: docs/commands/_category_.json (hunk 1 above)
Comment (craicoverflow) [reply docs/commands/_category_.json:1]: Sorry I don't fully understand your question.

Comment (wtrocki) [reply docs/commands/_category_.json:1]: Why file is removed vs we have that info in website. More of question so I will understand it.

Comment (craicoverflow) [reply docs/commands/_category_.json:1]: https://github.com/craicoverflow/app-services-cli/blob/ux-guidelines-base-structure/website/sidebars.js#L24-L30

I have added this to the sidebars file instead which does the same thing. I wanted to have the contributing section open by default and this one collapsed, and wanted more control over ordering.

Comment (wtrocki) [reply docs/commands/_category_.json:1]: Really cool! Thank you for explaining

**Pull request merged**